app/todolist.js

```diff
@@ -5,8 +5,10 @@
     this.items = itemsApi.items;
     this.item = {};
 
-    this.addItem = function(item) {
-      itemsApi.addItem(item);
+    this.addItem = function(itemDescription) {
+      if(description) {
```

Comment:
I think `description` is undefined here and you wanted to use `itemDescription` instead... right? Also this could be simplified a little using the early return approach.